ports/libunifex/vcpkg.json

```
@@ -1,7 +1,6 @@
 {
   "name": "libunifex",
-  "version-date": "2023-01-25",
-  "port-version": 2,
+  "version-date": "2023-06-07",
```

```suggestion
  "version": "0.3.0",
```
... would enable ...